### ISSUE TITLE
gthumb overlay select is opaque

### ISSUE THREAD

**Issue by DevArctic** [rtlewis1/GTK#1]
I use material-black pistachio and before the update when I would click in gthumb to select multiple files, the box that covers the files, like the selection area box, i guess, is now opaque where as before it would be like a transparent blue effect, which was nice. I don't think that's intended behavior on the theme's part.

**Comment by DevArctic** [comment]
for reference I'm using the latest version of the theme, on arch, using gthumb 3.12.2-2

**Comment by rtlewis1** [comment]
Gnome 4 based theming versus gnome 3 is a game changer on some theming elements, I noticed. It was exhausting to get it as close as I did. Let me see what the deal is on the pistachio before the next update. You're talking about, specifically, the image viewer and image organizer vis-a-vis gthumb?

**Comment by DevArctic** [comment]
yeah, when you click on a folder it brings up the images in the grid and you can select a bunch from there. the overlay color works in thunar, when using the same kinda feature.

**Comment by rtlewis1** [comment]
Okay, I may have figured out the the specific gthumb tweak you want. Give me a couple of more days, I've been dealing with cold-weather challenges at my property since Fri . I have to pin down the GTK 2/3/4 anomaly, wherein the gthumb selection config used to use some code, vis-a-vis the old MBC theme version, and make the newer theme architecture code available for the app to detect and isolate it specifically for use with the text and item selection. I'll let you know if I cracked it - and then make a file available for you asap if so.

**Comment by DevArctic** [comment]
no worries man. i ain't in a hurry.

**Comment by rtlewis1** [comment]
Pleases try the new (LA) version that was pushed yesterday, and make sure it brings back your expected user experience.
Best Regards

**Comment by DevArctic** [comment]
I have tried it every which way. I

**Comment by DevArctic** [comment]
I have tried it every which way. I reinstalled gthumb, I deleted all my themes, tried it in /usr/share/themes, then ~/.themes then I deleted and remade the gtk configs, i used lx appearance, i did it by hand, etc etc, gthumb still shows a completely opaque box when doing the drag tool

**Comment by rtlewis1** [comment]
You have tried this variant?
[Material-Black-Pistachio-LA_2.8.9.zip](https://github.com/rtlewis1/GTK/files/10422012/Material-Black-Pistachio-LA_2.8.9.zip)

**Comment by DevArctic** [comment]
Yes, that's exactly what I did. Unless there's something I'm missing, it just doesn't work on my system. Do you know if programs ever cache older themes or something?

**Comment by rtlewis1** [comment]
Your system (some apps) will only cache older themes until a session has ended. Logging in & out mostly solves that. Just the same, I am working on something for you now - I will be finished within the hour - and then I will post the file here. I am hoping I finally solved this for us.

**Comment by rtlewis1** [comment]
[Material-Black-Pistachio-DevArctic.zip](https://github.com/rtlewis1/GTK/files/10431125/Material-Black-Pistachio-DevArctic.zip)

**Comment by DevArctic** [comment]
Nope. I still have some of the older versions installed, and they work just fine in gthumb, but I upgraded because thunar wasn't correct. So whatever changed between the big update and those previous versions is what did it.

**Comment by rtlewis1** [comment]
Yep, the big upgrade jumped from GTK3 to GTK4, pretty major. If you you care to keep up with the efforts, on your behalf, I'll keep trying - and put the file here again for you.

**Comment by DevArctic** [comment]
sure i mean its not the end of the world, im sure youll figure it out
